pages/dkp/kommander/1.4/release-notes/index.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: layout.pug
+beta: true
+navigationTitle: Kommander v1.4 Release Notes
+title: Release Notes
+menuWeight: 0
+excerpt: View release-specific information for Kommander
+enterprise: false
+---
+
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Kommander&reg; version 1.4.0 Beta 0 was released on ?????.**
```

Comment:
```suggestion
**D2iQ&reg; Kommander&reg; version 1.4.0 Beta 0 was released on March 10, 2021.**
```

Reply:
3/10 is the kommander 1.3.1 patch release. This 1.4.0 beta has already been released, last friday with konvoy 1.8.0 beta

Reply:
```suggestion
**D2iQ&reg; Kommander&reg; version 1.4.0 Beta 0 was released on 26, February 2021.**
```